project.clj

```diff
@@ -20,15 +21,19 @@
             [lein-cljsbuild "1.1.7" :exclusions [org.clojure/clojure
                                                  org.clojure/clojurescript]]
             [lein-figwheel "0.5.14" :exclusions [org.clojure/clojure
-                                                 org.clojure/clojurescript]]]
+                                                 org.clojure/clojurescript]]
+            [com.github.clj-kondo/lein-clj-kondo "0.2.4" :exclusions [org.clojure/clojure
+                                                                      org.clojure/clojurescript]]]
+  :aliases {"clj-kondo-deps" ["clj-kondo" "--copy-configs" "--dependencies" "--parallel" "--lint" "$classpath"]
```

Comment:
It’s not clear to me what this alias is for.

Reply:
this alias takes care of copying the lint dependencies of clara itself and populating the cache with the linted dependencies (for development and CI) before linting clara-rules itself.